edx_exams/apps/api/v1/views.py

```diff
@@ -554,6 +561,44 @@ def post(self, request):
         return Response(data)
 
 
+class InstructorAttemptsListView(ExamsAPIView):
+    """
+    Endpoint for listing exam attempts. Used to provide data for the instructor
+    facing exam dashboard.
+
+    /instructor_view/attempts?exam_id=<exam_id>
```

Comment:
I used a unique path for this compared to `exam/attempts` because the data returned is different than the other attempt endpoints.